### PR TITLE
Use response-model object in AdminConsoleWebscript

### PR DESCRIPTION
**Pull request by tgeens**
Refactoring the admin-console webscript to return a typed response model.

Converted the export-registry configs in the .ftl - other components will follow on follow up PRs, to keep the PRs smaller

### CODE REVIEW

File: alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java [hunk 1]
```diff
@@ -48,12 +66,32 @@ public AdminConsoleWebScript(MeterRegistry meterRegistry, Properties globalPrope
         return model;
     }
 
-    private void includeRegistryProperties(Map<String, Object> model, final String registry) {
-        Map<String, String> relevantProps = globalProperties.stringPropertyNames().stream()
-                .filter(s -> s.startsWith(PROP_PREFIX_REGISTRY + registry))
-                .collect(Collectors.toMap(s -> s, globalProperties::getProperty));
+    @VisibleForTesting
```
Comment (kerkhofsd) [inline alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java:69]: Black magic?

**Review by kerkhofsd — APPROVED** [review]

File: alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java (hunk 1 above)
Comment (todorinskiz) [reply alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java:69]: Also see: https://jira.sonarsource.com/browse/RSPEC-5803

**Review by todorinskiz — APPROVED** [review]
The remark about `@VisibleForTesting` is non-blocking

File: alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java (hunk 1 above)
Comment (tgeens) [reply alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/webscripts/console/AdminConsoleWebScript.java:69]: Simply an annotation that this method could have been private, but then I would not be able to access that from the tests.

**Pull request merged**